docs/kratos/social-signin/10_google.mdx

```diff
@@ -219,3 +219,23 @@ Follow these steps to add Google as a social sign-in provider to your project us
 </TabItem>
 </Tabs>
 ````
+
+### Additional Parameters
+
+The following parameters are available for the Google provider:
+
+- `login_hint`
+- `hd`
+
+The `login_hint` parameter suppresses the account chooser and either pre-fills the email box on the sign-in form, or selects the
+proper session. The `hd` parameter limits the login/registration process to a Google Organization, e.g. `mycollege.edu`.
+
+The parameters can be passed along to Ory on `login`, `registration` and `settings` flows on form submit.
+
+```html
+<form action="https://{project.slug}.projects.oryapis.com/self-service/login?flow=<flow-id>" method="POST">
+  <input type="submit" name="provider" value="google" />
+  <input type="hidden" name="upstream_parameters.login_hint" value="foo@bar.com" />
+  <input type="hidden" name="upstream_parameters.hd" value="bar.com" />
+</form>
+```
```

Comment:
I don't think you really need to add this content here. The use case for this page is for a system administrator, who wants to enable Google social sign-in. But this new content is aimed at an app developer, who is customising the UI. Maybe you could just add a cross-reference to the other page, as in "See also ...".